integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/QuerySpecs.scala

```diff
@@ -61,14 +243,14 @@ class QuerySpecs(config: Config, spark: SparkSession) {
     }
   }
 
-  def getCandidateQueries(): Map[String, TestQuery] = {
+  def getCandidateQueries(): mutable.LinkedHashMap[String, TestQuery] = {
```

Comment:
why are the maps mutable? Do we plan on changing them after they are constructed?

Reply:
I was looking for Map that can reserve insertion order and found LinkedHashMap first. I just found ListMap can also do this, updated.